lnclient/ldk/ldk.go

```diff
@@ -1427,6 +1428,28 @@ func (ls *LDKService) publishChannelsBackupEvent() {
 		})
 	}
 
+	func() {
```

Comment:
why do we need this function here?

do you use it mainly because of the error handling/returns? 

wondering if it should/could be extracted into a individual function. or even a helper function that writes data to disc?

SaveBackup(backupDirectory, channels)

that this event handler does not need to know how to write files.

Reply:
Yeah, for consistency in how we handle errors. I extracted it to a separate method, thanks